packages/boundless-checkbox-group/style.styl

```diff
@@ -1,5 +1,5 @@
 @require "variables"
-@require "node_modules/boundless-checkbox/style";
+@require "../boundless-checkbox/style";
```

Comment:
I don't think this would work? If you `npm i boundless-checkbox-group`, that relative directory won't exist.